lib/chef/application/windows_service.rb

```diff
@@ -264,6 +266,14 @@ def auto_log_level?
         Chef::Config[:log_level] == :auto
       end
 
+      def resolve_log_location
+        if Chef::Config[:log_location] && Chef::Config[:log_location].is_a?(String)
```

Comment:
What happens if Chef::Config[:log_location] == STDOUT? (the path in the old code)